### PR TITLE
CADC-12092: single skaha/image web service call instead of one per session type

### PR DESCRIPTION
**Pull request by hjeeves**
as description - makes UI page load more efficient.

### CODE REVIEW

File: src/main/webapp/js/science_portal_form.js [hunk 1]
```diff
@@ -110,70 +110,60 @@
...
         })
     }
 
-    // Q: is the full session data being passed in or just the type?
-    //function getImageListForSessionType(sessionData) {
-    //  return getImageListForType(sessionData.name)
-    //}
-
-
     function getImageListForType(sessionType) {
```
Comment (yeunga) [inline src/main/webapp/js/science_portal_form.js:159]: Is this function still used somewhere?

**Review by yeunga — APPROVED** [review]
Looks good other than not sure if the getImageListForType(sessioinType) function is still needed.

File: src/main/webapp/js/science_portal_form.js (hunk 1 above)
Comment (ijiraq) [reply src/main/webapp/js/science_portal_form.js:159]: Yes, it's used here: 

https://github.com/opencadc/science-portal/blob/3a24e68334cd3b78419aaa95c6a5d4b491f20d28/src/main/webapp/js/science_portal.js#L591 

but perhaps that doesn't get called anymore?

Comment (yeunga) [reply src/main/webapp/js/science_portal_form.js:159]: That's good.

Comment (hjeeves) [reply src/main/webapp/js/science_portal_form.js:159]: Thanks, JJ for answering that - it is still used - at the point someone selects a new session type in the launch form, that function will be called to get the list of images to populate the image dropdown.

**Review by at88mph — APPROVED** [review]

**Pull request merged**